config/channel/resources/kafkachannel-crd.yaml

```diff
@@ -48,6 +48,9 @@ spec:
                   type: integer
                   maximum: 32767
                   default: 1
+                retentionMillis:
+                  description: retentionMillis is the topic retention period (in ms) of a Kafka Topic. This is an optional field. By default, the Channel default retention time provided in the ConfigMap would be used. If the default value is not provided in Config Map, 604800000(1 week) is used.
```

Comment:
Should we use the ISO8601 format instead? This is what we use in other eventing apis, for example in the DeliverySpec https://github.com/knative/eventing/blob/main/pkg/apis/duck/v1/delivery_types.go#L52